openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AddRoleAndAssignToUser.spec.js

```diff
@@ -98,15 +98,34 @@ describe("Test Add role and assign it to the user", () => {
     .should('exist')
     .should('be.visible')
     .type('Adding user');
+
+    interceptURL('GET', '/api/v1/users/generateRandomPwd', 'generatePassword');
+    cy.get('[data-testid="password-generator"]').scrollIntoView().should('be.visible').click();
+    verifyResponseStatusCode('@generatePassword', 200);
     
     cy.get(`[id="menu-button-Roles"]`).should("exist").should("be.visible").click()
 
-    cy.get(`[data-testid="${roleName}"]`).should("be.visible").click()
+    cy.get(`[data-testid="${roleName}"]`).scrollIntoView().should("be.visible").click()
+
+    cy.get('[data-testid="roles-dropdown"]').click();
+
+    cy.wait(1000);
```

Comment:
is it necessary?

Reply:
Its just a precaution since there are no api calls to wait on